test/studies/shootout/submitted/knucleotide-output.txt.good

```diff
@@ -0,0 +1,27 @@
+T 31.520
+A 29.600
+C 19.480
+G 19.400
+
+AT 9.922
+TT 9.602
+TA 9.402
+AA 8.402
+GA 6.321
+TC 6.301
+TG 6.201
+GT 6.041
+CT 5.961
+AG 5.841
+CA 5.461
+AC 5.441
+CC 4.041
+CG 4.021
+GC 3.701
+GG 3.341
```

Comment:
Have you checked this PR with the .perfkeys verification values?

Reply:
Good thought, I haven't.  You're not expecting it to break, though, are you?  (in that the behavior for get() should be the same when there are no missing keys?

Reply:
It shouldn't, but after our chat with the diff:

```
< C 19.773
---
> C 19.772

< TC 5.958
---
> TC 5.957
```

I may be a bit paranoid. Especially in the interest of getting performance results tonight.

Reply:
Looks clean as expected.